micrometer-commons/src/main/java/io/micrometer/common/lang/package-info.java

```diff
@@ -13,6 +13,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@NonNullApi
-@NonNullFields
```

Comment:
I suppose it may be better to make this `@NullMarked` even though we don't have any nullability in the annotations in this package.